client/components/NavBar.js

```diff
@@ -2,21 +2,29 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 
+import Notification from './Notification';
+
 export class NavBar extends React.Component {
   constructor(props){
     super(props);
 
     this.logOut = this.logOut.bind(this);
   }
-
+  /**
+   * [logOut description]
+   * @method logOut
+   * @return {[type]} [description]
+   */
   logOut() {

```

Comment:
Documentation could be improved a little, i think the valid return type for this function is `void`
`